WBooks/Library/LibraryViewController.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  LibraryViewController.swift
+//  WBooks
+//
+//  Created by Carolina Arcos on 8/21/18.
+//  Copyright © 2018 Wolox. All rights reserved.
+//
+
+import UIKit
+
+class LibraryViewController: UIViewController {
+    
+    // MARK: - Constants
+    
+    struct Constants {
+        static let backgroundBlueColor = UIColor(hex: "EAF6FA")
+        static let tablePositionY: CGFloat = 40
+        static let defultEdgeValue: CGFloat = 0
```

Comment:
If the only important margin is the top one, you can avoid the unnecessary constants like defaultEdgeValue, but it's okay if you want to leave it because it's useful if we change marging.

And also, I would like a better naming. For example, both of these are "margins" or "edges" (top and others) if you prefer, not one position and one edge.